jvm/src/common/Language/Java/Internal.hs

```diff
@@ -155,3 +160,20 @@ getStaticFieldAsJValue retsing cname fname = do
     SPrim "double" -> JDouble <$> getStaticDoubleField klass field
     SVoid -> fail "getStaticField cannot yield an object of type void"
     _ -> JObject <$> getStaticObjectField klass field
+
+mkVariadic :: TH.TypeQ -> (TH.TypeQ -> TH.TypeQ -> [TH.PatQ] -> TH.ExpQ -> TH.ExpQ -> TH.DecsQ) -> TH.DecsQ
+mkVariadic retty k = fmap concat $ for [0..32 :: Int] $ \n -> do
```

Comment:
Should 32 be a defined constant somewhere? Presumably, this number is the maximum number of arguments supported in variadic notation, and perhaps clients will want to query it.

Reply:
Indeed `inline-java` (which is a package downstream of `jvm`) currently duplicates the same constant (with a comment pointing here). I'll export a constant.